xml/ha_hawk2_history_i.xml

```diff
@@ -317,11 +317,13 @@
   <title>Viewing Transition Details in the History Explorer</title>
   <para>
    For each transition, the cluster saves a copy of the state which it provides
-   as input to the policy engine (PE). The path to this archive is logged. All
+   as input to <systemitem class="daemon">pacemaker-schedulerd</systemitem>. 
+   The path to this archive is logged. All
    <filename>pe-input*</filename> files are generated on the Designated
    Coordinator (DC). As the DC can change in a cluster, there may be
    <filename>pe-input*</filename> files from several nodes. Any
-   <filename>pe-input*</filename> files show what the PE
+   <filename>pe-input*</filename> files show what <systemitem
+    class="daemon">pacemaker-schedulerd</systemitem>
    <emphasis>planned</emphasis> to do.
```

Comment:
Any `pe-*` files are saved snapshots of the CIB used as input of calculations by pacemaker-schedulerd.

Reply:
Changed, thanks.